packages/env/lib/cli.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+/**
+ * External dependencies
+ */
+const chalk = require( 'chalk' );
+const ora = require( 'ora' );
+const yargs = require( 'yargs' );
+const terminalLink = require( 'terminal-link' );
+
+/**
+ * Internal dependencies
+ */
+const env = require( './env' );
+
+// Colors
```

Comment:
I like the idea of defining standard colours to use for CLI interfaces. This could be spun out into thing that could be used for all CLI scripts. (Not a blocker for this PR, though.)

Reply:
Yeah, that'd be nice, I got them from the `_colors.scss` file.